frontend/src/components/Settings/ProfileInfoSetting.js

```diff
@@ -0,0 +1,118 @@
+// Importing necessary modules
+import React, { useEffect, useState } from "react";
+import {
+	collection,
+	query,
+	where,
+	getDoc,
+	doc,
+	onSnapshot,
+	getDocs,
+	deleteDoc,
+	updateDoc,
+} from "firebase/firestore";
+import { auth, db } from "../../firebase";
+import Card from "react-bootstrap/Card";
+import "../../styles/JobPostings.css";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+
+import arrow from ".././../images/mdi_arrow.png";
+
+import { Link, useNavigate } from "react-router-dom";
+function ProfileInfoSettings() {
+
+
+	// Function to redirect to the "Account Preferences" page
+	const handleClickAccount = () => {
+		window.location.href = "/ProfileInfoSettings";
+	};
+
+	// Function to redirect to the "JobPostings" page
+	const handleClickSecurity = () => {
+		window.location.href = "/Security";
+	};
+	const handleClickNotifications = () => {
+		window.location.href = "/";
+	};
+
+	
+	// This component displays a page for job postings
+	// and advertisements with a menu block on the left to navigate between them.
+
+	return (
+		<Container className="container d-flex justify-content-center mx-auto">
+			{/* JOB MENU BLOCK ON THE LEFT TO NAVIGATE BETWEEN JOB POSTINGS AND ADVERTS */}
+			<Row
+				className="gap-6 d-flex justify-content-center"
+				style={{ minWidth: "80%" }}
+			>
+				<Col xs={12} sm={8} lg={4} style={{ minWidth: "30%" }}>
```

Comment:
The buttons could use a hover effect, but the functionalities are there, nice work!